### PR TITLE
PXC-4161 Document wsrep_mode variable (8.0.31-23)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/wsrep-system-index.md

### CODE REVIEW

File: docs/wsrep-system-index.md [hunk 1]
```diff
@@ -781,6 +781,26 @@ You can set it to any value between `1024` and the default `2147483647`.
...
+
+This variable has been implemented in [Percona XtraDB Cluster 8.0.31](./release-notes/8.0.31-23.md).
+
+Defines the node behavior according to a specified value. The variable is disabled by default.
+
+The available value is `IGNORE_NATIVE_REPLICATION_FILTER_RULES` which makes `WSREP` to ignore native replication filter rules.
+
+!!! admonition "See also"
+
+    [MySQL wsrep option: wsrep_mode](https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_mode)
```
Comment (patrickbirch) [inline docs/wsrep-system-index.md]: use
https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-mode

File: docs/wsrep-system-index.md [hunk 2]
```diff
@@ -781,6 +781,26 @@ You can set it to any value between `1024` and the default `2147483647`.
...
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep-mode``   |
+| Config File:   | Yes                |
+| Scope:         | Global             |
+| Dynamic:       | Yes                |
+| Default Value: |                    |
+
+This variable has been implemented in [Percona XtraDB Cluster 8.0.31](./release-notes/8.0.31-23.md).
+
+Defines the node behavior according to a specified value. The variable is disabled by default.
```
Comment (patrickbirch) [inline docs/wsrep-system-index.md]: It has two possible values: empty and IGNORE_NATIVE_REPLICATION_FILTER_RULES
So - does empty = disabled?

File: docs/wsrep-system-index.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/wsrep-system-index.md]: updated

File: docs/wsrep-system-index.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/wsrep-system-index.md]: updated

File: docs/wsrep-system-index.md [hunk 3]
```diff
@@ -781,6 +781,30 @@ You can set it to any value between `1024` and the default `2147483647`.
...
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep-mode``   |
+| Config File:   | Yes                |
+| Scope:         | Global             |
+| Dynamic:       | Yes                |
+| Default Value: |                    |
+
+This variable has been implemented in [Percona XtraDB Cluster 8.0.31](./release-notes/8.0.31-23.md).
+
+Defines the node behavior according to a specified value. Empty by default.
```
Comment (patrickbirch) [inline docs/wsrep-system-index.md]: The value is empty or disabled by default.

File: docs/wsrep-system-index.md [hunk 4]
```diff
@@ -781,6 +781,30 @@ You can set it to any value between `1024` and the default `2147483647`.
...
+
+The available values are:
+
+* `Empty` - does not change the node behavior.
+
+* `IGNORE_NATIVE_REPLICATION_FILTER_RULES` - makes `WSREP` to ignore native replication filter rules.
```
Comment (patrickbirch) [inline docs/wsrep-system-index.md]: Changes the wsrep behavior to allow ignoring the native replication filter rules.

File: docs/wsrep-system-index.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/wsrep-system-index.md]: updated

File: docs/wsrep-system-index.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/wsrep-system-index.md]: Updated

**Review by patrickbirch — APPROVED** [review]
good

**Review by kamil-holubicki — APPROVED** [review]

File: docs/wsrep-system-index.md [hunk 5]
```diff
@@ -781,6 +781,30 @@ You can set it to any value between `1024` and the default `2147483647`.
...
+
+This variable has been implemented in [Percona XtraDB Cluster 8.0.31](./release-notes/8.0.31-23.md).
+
+Defines the node behavior according to a specified value. The value is empty or disabled by default.
+
+The available values are:
+
+* `Empty` - does not change the node behavior.
+
+* `IGNORE_NATIVE_REPLICATION_FILTER_RULES` - changes the `wsrep` behavior to allow ignoring the native replication filter rules.
```
Comment (dutow) [inline docs/wsrep-system-index.md]: maybe "behavior to ignore native replication filter rules"?

Comment (alina-derkach-oaza) [reply docs/wsrep-system-index.md]: updated

**Pull request merged**